integrations/huggingface-transformers/tutorials/sentiment-analysis/sentiment-analysis-cli.md

```diff
@@ -0,0 +1,328 @@
+      final_lr: eval(final_lr)
+
+quantization_modifiers:
+  - !QuantizationModifier
+      start_epoch: eval(qat_start_epoch)
+      disable_quantization_observer_epoch: eval(observer_epoch)
+      freeze_bn_stats_epoch: eval(observer_epoch)
+      quantize_embeddings: eval(quantize_embeddings)
+      quantize_linear_activations: 0
+      exclude_module_types: ['LayerNorm', 'Tanh']
+      submodules:
+        - bert.embeddings
+        - bert.encoder
+        - bert.pooler
+        - classifier
+
+distillation_modifiers:
+  - !DistillationModifier
+     hardness: eval(distill_hardness)
+     temperature: eval(distill_temperature)
+     distill_output_keys: [logits]
+
+constant_modifiers:
+  - !ConstantPruningModifier
+      start_epoch: 0.0
+      params: __ALL_PRUNABLE__
+
+regularization_modifiers:
+  - !SetWeightDecayModifier
+      start_epoch: 0.0
+      weight_decay: eval(weight_decay)
+```
+
+</details>
+
+You can download the recipe with the following code:
+
+```python
+from sparsezoo import Model
+transfer_stub = "zoo:nlp/sentiment_analysis/obert-base/pytorch/huggingface/sst2/pruned90_quant-none"
+download_dir = "./transfer_recipe"
+zoo_model = Model(transfer_stub, download_path=download_dir)
+recipe_path = zoo_model.recipes.default.path
+print(recipe_path)
+```
+
+### **Export to ONNX**
+
+Once you have trained your model, export to ONNX in order to deploy with DeepSparse. The artifacts of the training process 
+are saved to your local filesystem. 
+
+Run the following to convert your PyTorch checkpoint to ONNX:
+
+```bash
+sparseml.transformers.export_onnx \
+  --model_path sparse_quantized_bert-text_classification_sst2 \
+  --task text_classification
+```
+
+A `deployment` folder is created in your local directory, which has all of the files needed for deployment with DeepSparse including the `model.onnx`, `config.json`, and `tokenizer.json` files.
+
+## Sparse Transfer Learning with a Custom Dataset (Rotten Tomatoes)
+
+Beyond the built-in GLUE tasks, we can also use a dataset from the Hugging Face Hub or pass via local files. Let's try an example of each for the sentiment analysis using [Rotten Tomatoes Dataset](https://huggingface.co/datasets/rotten_tomatoes), which containing 5,331 positive and 5,331 negative processed sentences.
+
+For simplicity, we will perform the fine-tuning without distillation. Although the transfer learning recipe contains distillation
+modifiers, by setting `--distill_teacher disable` we instruct SparseML to skip distillation.
+
+### Rotten Tomatoes Inspection
+
+Run the following to inspect the Rotten Tomatoes dataset.
+
+```python
+from datasets import load_dataset
+
+rotten_tomatoes = load_dataset("rotten_tomatoes")
+print(rotten_tomatoes)
+print(rotten_tomatoes["train"][0])
+```
+
+We can see that each row dataset contains a `text` field which is a string representing the sequence to be classified and a `label` field which is a `0` or `1` representing negative and positive labels.
```

Comment:
We can see that each row dataset contains a `text` field which is a string representing the sequence to be classified, and a `label` field which is a `0` or `1` representing negative and positive labels.